Source/santa-driver/SantaCache.h

```diff
@@ -166,6 +182,20 @@ template<class T> class SantaCache {
   }
 
   /**
+    Overload to allow setting without providing a previous value
+  */
+  T set(uint64_t key, T value) {
+    return set(key, value, {}, false);
```

Comment:
That's C++11 list initialization, it has fewer limitations on provided types. It doesn't make much difference in our use but it's a nice switch.